javascript/javascript2/week2/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>JavaScript2 Week2 HomeWork</title>
+    <link rel="stylesheet" href="/week2/styles.css" />
```

Comment:
Please check the path to styles.css
Being index.html, script.js and styls.css at the same level, the path should not include "/week2".